project_internal_access_from_portal/tests/test_portal_internal_access.py

```
@@ -0,0 +1,77 @@
+from odoo.addons.project.tests.test_access_rights import TestAccessRights
+
+
+class TestPortalInternalAccess(TestAccessRights):
+    """
+    Tests for the `portal_internal` visibility mode:
+    - Portal user may read only when subscribed
+    - Portal user cannot write/create/unlink projects and tasks
+    """
+
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.8.2)

Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)